elements/lisk-p2p/test/unit/p2p.ts

```diff
@@ -75,24 +82,59 @@ describe('p2p', () => {
 			);
 		});
 
-		it('should configure Websocket options', async () => {
-			const websocketOptions = (P2PNode as any)._scServer.wsServer.options;
-
-			expect(websocketOptions).toMatchObject({
-				maxPayload: DEFAULT_WS_MAX_PAYLOAD / 2,
-			});
-		});
-
 		it('should reject at multiple start attempt', async () => {
-			await P2PNode.start();
-
 			expect(P2PNode.start()).rejects.toThrow();
 		});
 
 		it('should reject at multiple stop attempt', async () => {
 			await P2PNode.stop();
-
 			expect(P2PNode.stop()).rejects.toThrow();
 		});
 	});
+
+	describe('#PeerServer', () => {
```

Comment:
it looks unusual that we are having a unit test based on real sockets and still mocking some functions of PeerPool. Need to think about making it consistent